### ISSUE TITLE
‘No Response’ accessories in Home if Hoobs Web Ui closed

### ISSUE THREAD

**Issue by sterumbelow** [hoobs-org/HOOBS#717]
Hi, everyone. 

I’ve just got my very first Raspberry Pi after discovering Hoobs a couple of days ago, realising it’ll enable to many devices (including my SimpliSafe alarm) to work with Home. 

I got the RPi 4, downloaded the latest version of Hoobs, flashed the card, set it all up and it works.

However, my accessories quite quickly say ‘no response’. I’ve been pulling my hair out slightly trying to understand why this would be happening and have found the following pattern:

As soon as my laptop is closed, the service stops and the sensors go offline. When I reopen my laptop and go to the Hoobs WebUI, it immediately comes back online.

I’m unsure how this has ended up being related to the WebUI being open? Shouldn’t it run solely on the RPi? I followed the instructions to the letter so not sure what I’ve missed or done wrong!

Thanks so much for your help,
Ste. 

****EDIT - RESOLVED**** 

My ‘no response’ problems were caused by connecting Hoobs/Homebridge/RPi to the 5Ghz band of my router. SimpliSafe connects to 2.4Ghz bands only, so as I switched over it has worked flawlessly. 

TL:DR If you’re using SimpliSafe with Homebridge/Hoobs, make sure you’re connecting to a 2.4Ghz band.

**Comment by sterumbelow** [comment]
Annoyingly, I’ve just slightly disproven my own theory. 

I just went out for 45 minutes or so. The sensors were not response’ status after a few minutes. Then when I arrived home, the Home app tried to run an automation (disable alarm when I get home) and it seemed to pull everything back online. 

I’m at a total loss to understand what’s going on!

**Comment by sterumbelow** [comment]
Update to this:

The behaviour seems to have stopped now and everything stays online for the most part. Then sometimes, randomly, the system goes offline for a bit with ‘no response’ and then comes back online by itself. 

Any idea what would cause that?

**Comment by sterumbelow** [comment]
**RESOLVED** 

My ‘no response’ problems were caused by connecting Hoobs/Homebridge/RPi to the 5Ghz band of my router. SimpliSafe connects to 2.4Ghz bands only, so as I switched over it has worked flawlessly. 

TL:DR If you’re using SimpliSafe with Homebridge/Hoobs, make sure you’re connecting to a 2.4Ghz band.
